build/Dockerfile.base

```diff
@@ -78,13 +79,39 @@ RUN groupadd --gid 53150 -r easy_infra \
  && rm -f /usr/local/bin/envconsul.zip \
  && chmod 0755 /usr/local/bin/envconsul \
  && apt-get remove -y unzip \
- && apt-get install -y --no-install-recommends cmake build-essential flex bison libyaml-dev libssl-dev \
+ && apt-get install -y --no-install-recommends cmake \
+                                               build-essential \
+                                               flex \
+                                               bison \
+                                               libyaml-dev \
+                                               libz-dev \
+                                               libssl-dev \
+                                               libcurl4-gnutls-dev \
+                                               libexpat1-dev \
+                                               make \
+                                               gcc \
+ # Build a new version of the git binary to support loading environment variables to designate safe directories
```

Comment:
Please include specific pointers to the minimum version needed and the git commit on git/git explaining why